EpiAware/test/EpiObsModels/Ascertainment.jl

```diff
@@ -0,0 +1,23 @@
+@testitem "Test Ascertainment constructor" begin
+    using Turing
+    function natural(x)
```

Comment:
Why not `x -> x`?

Reply:
Ah is it to check `==` at the bottom?

Reply:
because I am basic!

Reply:
THis being said we don't want to address this here right as the change here was just to take out the use of scale?